editor/utils/tracking.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
Maybe the tracking could be its own module, it could be useful outside of the editor (and other WP feature plugins later)